### ISSUE TITLE
Phinx migrations dies on fresh install stap_3

### ISSUE THREAD

**Issue by JJKalnins** [copona/copona#197]
Older mysql databases don't support full text indexes
~~~
/home/www/website.com/www/public_html/vendor/robmorgan/phinx/src/Phinx/Db/Adapter/MysqlAdapter.php(570):
Phinx\\Db\\Adapter\\PdoAdapter->execute('ALTER TABLE `cp...')
~~~
~~~
/home/www/website.com/www/cgi-bin/php5-fcgi" stderr: #3 
/home/www/website.com/www/public_html/vendor/robmorgan/phinx/src/Phinx/Db/Adapter/TablePrefixAdapter.php(190):
 Phinx\\Db\\Adapter\\AdapterWrapper->addIndex(Object(Phinx\\Db\\Table), 
Object(Phinx\\Db\\Table\\Index))
~~~
~~~
General error: 1214 The used table type doesn't support FULLTEXT indexes' in 
/home/www/website.com/www/public_html/vendor/robmorgan/phinx/src/Phinx/Db/Adapter/PdoAdapter.php:338,
 referer: http://website.com/install/index.php?route=install/step_3
~~~

There should be some kinda error handle for this

**Comment by arnisjuraga** [comment]
@prhost or @lucasjkr , can You, please, add MySQL version check?
In fact, this would not be a problem to be installed on older MySQLs, (those indexes does not impact CP installation. Those indexes yet are used only by extensions (with `MATCH()` selects.

But, if this can't be ignored during the installation, then limitations must be applied during install.

**Comment by lucasjkr** [comment]
I'll work on the version check, I need it too.

I don't think you should ignore the version though (which I'm assuming you might be thinking about since you're saying it won't have impact? maybe I'm wrong)

**Comment by arnisjuraga** [comment]
Yes, this was my first thoughts. But of course, it shouldn't. :)

**Comment by lucasjkr** [comment]
One issue I realized. You can't check the MySQL version as a warning during install (like, where it says what PHP modules are installed), because you can't communicate with the DB to check the version until AFTER you've set up the connection.

Also, you're still pretending to support other databases (MSSQL, Postgres). You should officially end that support if you're going to do this through SQL. Otherwise, wait for @prhost to show off his ORM, and have that do the checking against the DBs.
